static/app/components/events/eventReplay/replayClipPreview.tsx

```diff
@@ -207,9 +208,10 @@ function ReplayClipPreview({
 
   return (
     <ReplayContextProvider
+      analyticsContext={analyticsContext}
       isFetching={fetching}
+      prefsStrategy={StaticReplayPreferences()}
       replay={replay}
```

Comment:
@malwilley i noticed that we are not setting `initialTimeOffsetMs` here, should we make it so the replay starts at the time of the error?